src/js/components/Tiles.js

```diff
@@ -30,9 +34,16 @@ export default class Tiles extends Component {
     this._layout = this._layout.bind(this);
     this._onClick = this._onClick.bind(this);
 
+    const tileSize = this.props.size;
+    const minColumnWidth = (tileSize === 'small') ? MIN_COLUMN_SIZE_SMALL :
+      ((tileSize === 'large') ? MIN_COLUMN_SIZE_LARGE : MIN_COLUMN_SIZE);
+    this._minColumnWidths = Array.apply(null, Array(this.props.numColumns))
+      .map((currentNumColumns, index) => (index + 1) * minColumnWidth);
```

Comment:
This reads like the width of the columns is variable based on in the index across. If I'm understanding this right, should the outer variable be something more like `columnOffsets`? Or am I missing something?